Modules/Common/src/TH2Ratio.cxx

```diff
@@ -0,0 +1,228 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file TH2Ratio.cxx
+/// \brief An example of a custom TH2Quotient inheriting MergeInterface
+///
+/// \author Piotr Konopka, piotr.jan.konopka@cern.ch, Sebastien Perrin, Andrea Ferrero
+
+#include "Common/TH2Ratio.h"
+#include "QualityControl/QcInfoLogger.h"
+#include <TList.h>
+
+#include <fmt/format.h>
+#include <limits>
```

Comment:
```suggestion
```
I don't see them used, could you please remove them to reduce the compilation time?